src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dailyCosts = 40;
+  const type1DayDiscount = 20;
+  const type2DayDiscount = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const largeDiscount = 50;
```
Цифри 1 та 2 нічого нам не кажуть. Назви змінних повинні бути такими, щоб несли в собі якийсь сенс